action-server/setup.cfg

```diff
@@ -0,0 +1,48 @@
+[tool:pytest]
```

Comment:
@dmorand Can you review what's in there? I've mostly took what's from beautiful-mind.